### PR TITLE
feat(TNLT-3680): front lead one and one slice

### PR DESCRIPTION
**Pull request by c-j-j**
Simulator Screen Shot - iPad Pro (12 9-inch) (4th generation) - 2020-08-20 at 11 47 01
Simulator Screen Shot - iPad Pro (12 9-inch) (4th generation) - 2020-08-20 at 11 46 34
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-08-20 at 11 19 39
Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2020-08-20 at 11 19 34
Simulator Screen Shot - iPad mini 4 - 2020-08-20 at 11 13 10
Simulator Screen Shot - iPad mini 4 - 2020-08-20 at 11 13 08

- [x] Portrait View
- [x] Landscape View
- [x] Tests
- [x] Stories

### CODE REVIEW

File: packages/edition-slices/edition-slices.showcase.js [hunk 1]
```diff
@@ -177,6 +178,12 @@ const sliceStories = [
     Slice: LeadTwoNoPicAndTwoFrontSlice,
     scroll: false,
   },
+  {
+    mock: mockLeadOneAndOneSlice(),
```
Comment (c-j-j) [inline packages/edition-slices/edition-slices.showcase.js]: I need to remove

File: packages/edition-slices/src/tiles/tile-a-front/index.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  const crop =
+    breakpoint === "huge" || orientation === "portrait" ? "crop32" : "crop169";
+  const showSummary = orientation === "portrait";
+
+  const imageCrop = getTileImage(tile, crop);
+  const styles = stylesFactory(breakpoint);
+
+  if (!imageCrop) {
+    return null;
+  }
```
Comment (Geoff-Ford) [inline packages/edition-slices/src/tiles/tile-a-front/index.js:34]: one liner

File: packages/edition-slices/src/tiles/tile-a-front/index.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+  const {
+    article: { hasVideo },
+  } = tile;
+
+  const summary = tile.article.content;
```
Comment (Geoff-Ford) [inline packages/edition-slices/src/tiles/tile-a-front/index.js:40]: Any reason for this?

**Review by Geoff-Ford — APPROVED** [review]

File: packages/edition-slices/src/tiles/tile-a-front/index.js (hunk 3 above)
Comment (c-j-j) [reply packages/edition-slices/src/tiles/tile-a-front/index.js:40]: Nope - I can inline. It is unnecessary.

Comment (c-j-j) [reply packages/edition-slices/src/tiles/tile-a-front/index.js:40]: Actually - I'll do the same as the other tile, which is to pull out `article` from `tile` and then use that afterwards.

**Pull request merged**